libraries/ms-drivers/test/test_bts7040_load_switch.c

```diff
@@ -365,6 +365,88 @@ void test_bts7040_current_sense_get_measurement_pca9539r_valid(void) {
   LOG_DEBUG("PCA9539R reading: %d\n", reading);
 }
 
+// Test that bts7040_get_measurement with the bts7004 scaling factor returns a different reading
+// than normal
+void test_bts7004_current_sense_get_measurement_stm32_valid(void) {
+  // this doesn't matter (adc isn't reading anything) but can't be null
+  GpioAddress test_sense_pin = { .port = GPIO_PORT_A, .pin = 0 };
+  GpioAddress test_enable_pin = { .port = GPIO_PORT_A, .pin = 1 };  // EN pin
+  uint32_t interval_us = 500;                                       // 0.5 ms
+  Bts7040Stm32Settings bts7004settings = {
```

Comment:
Nitpick: could you change these names like `btsXXXXsettings` to `btsXXXX_settings` (both here and in test_output.c)? That way they better comply with snake_case and look less smushed together (we tend to treat `bts7040` as one word for snake_case purposes).